.github/workflows/ci.yml

```diff
@@ -89,6 +90,18 @@ jobs:
         echo "CLANG_VERSION=$CLANG_VERSION" >> $GITHUB_ENV
         sudo apt-get install gcovr ${{ matrix.platform.name == 'Linux Clang' && 'llvm-$CLANG_VERSION' || '' }}
 
+    - name: Install Intel oneAPI
+      if: matrix.platform.name == 'Linux Intel oneAPI'
+      run: |
+        wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+        sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
```

Comment:
CI warns this is a deprecated command.
Looking it up online, I found an example using more secure commands: https://brave.com/linux/#debian-ubuntu-mint

Reply:
For posterity, here's the docs link Kimci shared with me in Discord:

https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-linux/2023-1/apt.html

I switched over to this approach and it seems to be working!